data/__generated__/schema.graphql

```diff
@@ -164,6 +164,30 @@ input ImageCreateInput {
   metadata: JSON
 }
 
+input ImageCreateManyInput {
+  images: [ImageCreateManySingleImage!]!
+  datasetId: ID!
+}
+
+input ImageCreateManySingleImage {
+  id: ID
+  createdAt: DateTime
```

Comment:
This should be set on server-side.

Reply:
See other comment, that's what we use to keep images in the same order as when they were uploaded.